### ISSUE TITLE
deploy fails on empty description

### ISSUE THREAD

**Issue by ijin** [marcy-terui/lamvery#43]
deploy fails even if the `description` parameter is `null` or empty in `.lamvery.yml`.

Since Lambda allows us to have an empty description (at least through the AWS console), I think this should be supported.
```
ERROR:root:Parameter validation failed:
Invalid type for parameter Description, value: None, type: <type 'NoneType'>, valid types: <type 'basestring'>
```

**Comment by marcy-terui** [comment]
Thank you for your reporting.

> Since Lambda allows us to have an empty description (at least through the AWS console), I think this should be supported.

I think so too.
It is a minor bug. So, I'll fix it in the near future.

**Comment by marcy-terui** [comment]
Done. It is released in [v0.12.1](https://pypi.python.org/pypi/lamvery/0.12.1)

Please check the new version :-)

**Comment by ijin** [comment]
confirmed fix. thanks! :+1:
